apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/Indictment.tsx

```
@@ -84,6 +82,30 @@ export const getIndictmentIntroductionAutofill = (
+]
+
+const getSuspensionOffences = (
+  indictmentCounts?: TIndictmentCount[] | null,
+) => {
+  const offenses =
+    indictmentCounts?.map(
+      (indictmentCount) =>
+        indictmentCount.offenses?.reduce<IndictmentCountOffense[]>(
+          (acc, offense: Offense) =>
+            suspensionOffenses.includes(offense.offense)
+              ? [...acc, offense.offense]
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)